cmd/git-sync/main.go

```diff
@@ -77,15 +76,22 @@ var flMaxSyncFailures = flag.Int("max-sync-failures", envInt("GIT_SYNC_MAX_SYNC_
 var flChmod = flag.Int("change-permissions", envInt("GIT_SYNC_PERMISSIONS", 0),
 	"the file permissions to apply to the checked-out files (0 will not change permissions at all)")
 var flSyncHookCommand = flag.String("sync-hook-command", envString("GIT_SYNC_HOOK_COMMAND", ""),
+	"(deprecated) the command executed with the syncing repository as its working directory after syncing a new hash of the remote repository. "+
```

Comment:
The v4 branch set a convention here.  Let's do `"DEPRECATED: use --exechook-command instead"`

Reply:
changed description.